js/packages/screenshoter/test/e2e/web/hover-element.spec.js

```diff
@@ -0,0 +1,27 @@
+const {makeDriver, test, logger} = require('../e2e')
+
+describe('screenshoter web', () => {
+  let driver, destroyDriver
+
+  before(async () => {
+    ;[driver, destroyDriver] = await makeDriver({type: 'chrome', logger})
+    await driver.visit('https://applitools.github.io/demo/TestPages/StickyHeaderWithRegions')
+  })
+
+  after(async () => {
+    await destroyDriver()
+  })
+
+  it('take element in frame screenshot with hover element', async () => {
```

Comment:
I would rather call it
```suggestion
  it('take hovered element screenshot', async () => {
```